### ISSUE TITLE
Dev sync reminder Github action

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/Organization#4]
As discussed in the most recent dev sync, it would be great to have a GitHub action that sends a message to remind people about the dev sync. We could use [s3krit/Matrix-Message](https://github.com/s3krit/Matrix-Message) for this. This would ideally be sent the Wednesday before the sync (no preference for time). A template message could be something along the following:

```
Hey all! 🤖👋 Here's the reminder for [this Saturday's dev sync at 16:00 UTC](https://zonestamp.toolforge.org/EPOCH_TIME) 🧑‍💻♻️

Details for it:

- Call link: https://call.element.io/room/#/activist-dev-sync?roomId=!UddhHUSXxHAoAnImXb:call.ems.host&password=ASriEQCG4DE6Q1QSB313zig0bhLd62RN
- Pad for this week: https://etherpad.wikimedia.org/p/activist-dev-sync-DATE-OF-SYNC-YYYY-MM-DD
- All dev sync pads: https://etherpad.wikimedia.org/p/activist-dev-sync

Please reply in the thread for this message with anything you'd like to discuss 🧵

Thanks and have a great day! ❤️
```

Welcome to discuss what else could be included!

**Comment by andrewtavis** [comment]
CC @wkyoshida, @to-sta, @momanyisamuel

**Comment by andrewtavis** [comment]
Note that I'm unsure about the suggested action at this point. Others to consider are:
- [select/matrix-message-e2e](https://github.com/select/matrix-message-e2e)
- [fadenb/Matrix-Chat-Message](https://github.com/fadenb/Matrix-Chat-Message)

**Comment by andrewtavis** [comment]
Closed by 351cafd 😊 Thanks, axif0!

**Comment by andrewtavis** [comment]
Really appreciate the work here, @axif0. I used the Scribe one here, and both of these together is really going to help my workflows :)
